src/core/appgateway.tf

```diff
@@ -313,6 +345,13 @@ module "app_gw" {
       priority              = 60
     }
 
+    continua-io-pagopa-it = {
```

Comment:
Why we need a new listener? this causes to maintain a new SSL certificate

Reply:
@pasqualedevita because we have to listen for `continua.io.pagopa.it`

Reply:
In this case we are not exposing APIs, but and endpoint for QR code rendering and and HTTP page for dynamic link matching

Reply:
The pipeline for a new certificated has already been PRed at pagopa/gitops#343

Reply:
However, if you can propose a better solution, feel free to reach us, we're open to consider that!

Reply:
can we reuse `api.io.pagopa.it` with a new product and api (api.io.pagopa.it/continua) without authentication?